IdentityManager.sln

```diff
@@ -1,45 +1,54 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Why IdentityManager files are are changed? Please revert them

Reply:
I do not know why git is saying that there is change here I even copped it from identity-manager/dev and pushed it